bin/docpub.js

```diff
@@ -10,18 +10,38 @@
 
 const program = require('commander');
 const pkg = require('../package.json');
-const DocpubPipeline = require('../lib/docpub');
+const Docpub = require('../lib/docpub');
 
 program
     .version(pkg.version)
-    .option('-v --verbose', 'verbose output')
-    .option('-p --path <path>', 'path to documents directory (defaults to process directory)');
+    .option('--doc-path <docPath>', 'path to documents directory (defaults to process directory)')
+    .option('--config-path <configPath>', 'path for the config')
+    .option('-v --verbose', 'verbose output');
+
+program.on('--help', () => {
+    console.log('  Overriding config');
+    console.log('    To override any config option use full option path converted to --kebab-case');
+    console.log('');
+
+    console.log('    Examples:');
+    console.log('      docpub --username user@example.com');
+    console.log('      docpub --token abcdefghijklmn12345');
+    console.log('');
+    console.log('    You can also use environment variables converted to snake_case with');
```

Comment:
My opinion is separate the verbose for `--help` will give us the chance to include the how to use the tool and it could be as big as the tool needs.  The theory is : 

- User read the documentation and know how to use the tool.
- User can get the `-h` to get the "cheat sheet" of the CLI that the tool uses. 
- User didn't read the doc, or don't want to go back, then `--help` will have more information and examples in how to use the tool , in this case , how to override the configuration file. 

For now is a small piece of information, but as we can add more features, this will be helpful for the command line user.